docs/embeddings.md

```diff
@@ -171,23 +220,35 @@ cohere_ef(texts=["document1","document2"])
 <TabItem value="js" label="JavaScript">
 
 ```javascript
-const {CohereEmbeddingFunction} = require('chromadb');
-const embedder = new CohereEmbeddingFunction("apiKey")
-
-// use directly 
-const embeddings = embedder.generate(["document1","document2"])
+const { CohereEmbeddingFunction } = require('chromadb');
+const embedder = new CohereEmbeddingFunction('apiKey');
 
 // pass documents to query for .add and .query
-const collection = await client.createCollection({name: "name", embeddingFunction: embedder})
-const collectionGet = await client.getCollection({name:"name", embeddingFunction: embedder})
+const collection = await client.createCollection({
+	name: 'name',
+	embeddingFunction: embedder,
+});
+const collectionGet = await client.getCollection({
+	name: 'name',
+	embeddingFunction: embedder,
+});
+
+// use directly
+const embeddings = await embedder.generate(['document1', 'document2']);
+
+// Add documents to the collection along with their associated embeddings.
+// Chroma will store these documents without generating its own embeddings.
+await collection.add({
```

Comment:
nit: Does it make sense also to include an example of `add()` without embeddings (e.g. leave the embedding to Chroma)